package.json

```diff
@@ -29,6 +29,8 @@
   },
   "private": true,
   "dependencies": {
+    "@actions/core": "^1.2.3",
```

Comment:
Should be dev deps and should have exact numbers. Remember to always use `npm i  lib-name --save-exact`